default.nix

```diff
@@ -75,28 +75,20 @@ let
         )
       ];
 
-  mkPgtkEmacs = namePrefix: jsonFile: { ... }@args: (mkGitEmacs namePrefix jsonFile args).overrideAttrs (
-    old: {
-      configureFlags = (super.lib.remove "--with-xft" old.configureFlags)
-        ++ super.lib.singleton "--with-pgtk";
-    }
-  );
-
-  emacsGit = mkGitEmacs "emacs-git" ./repos/emacs/emacs-master.json { withSQLite3 = true; withWebP = true; };
+  emacsGit = mkGitEmacs "emacs-git" ./repos/emacs/emacs-master.json { nativeComp = false; };
```

Comment:
Why do you disable nativeComp? I think it's better to be in line with nixpkgs, i.e. letting nixpkgs to decide enabling or disabling nativeComp by default.

Reply:
Should we instead define a new attribute `emacsGitNoNativeComp`? We would need to swap the naming for the other Emacsen as well.

Reply:
I do not think `noNativeComp` variants should be provided by this overlay.

To be in line with nixpkgs's change of enabling nativeComp by default and maintain backward compatibility, I suggest we do nothing for now and deprecate those nativeComp variants like [this](https://github.com/nix-community/emacs-overlay/blob/62c2d63c4fcb29719d2d4f722010c671d4497814/default.nix#L184) when that change reaches all nixpkgs versions this overlay support (I have no idea what they are).